gcp-compute-persistent-disk-csi-driver-1.15.yaml

```diff
@@ -0,0 +1,89 @@
+package:
+  name: gcp-compute-persistent-disk-csi-driver-1.15
+  version: 1.15.1
+  epoch: 0
+  description: The Google Compute Engine Persistent Disk (GCE PD) Container Storage Interface (CSI) Storage Plugin.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - blkid
+      - blockdev
+      - btrfs-progs
+      - busybox
+      - ca-certificates
+      - dbus-libs
+      - device-mapper
+      - e2fsprogs
+      - e2fsprogs-extra
+      - e2fsprogs-libs
+      - glibc
+      - json-c
+      - kmod-libs
+      - libacl1
+      - libblkid
+      - libbsd
+      - libcap
+      - libcom_err
+      - libcrypto3
+      - libgcc
+      - libgcrypt
+      - libgpg-error
+      - liblz4-1
+      - libmd
+      - libmount
+      - libnvme-dev
+      - libpcre2-8-0
+      - libstdc++
+      - libsystemd
+      - libuuid
+      - mount
+      - ncurses
+      - nvme-cli
+      - readline
+      - systemd
+      - umount
+      - userspace-rcu
+      - util-linux
+      - util-linux-misc
+      - xfsprogs
+      - xz
+      - zlib
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver
+      tag: v${{package.version}}
+      expected-commit: 8c21c23edf516250992f2b9a0a2cd667489d30b8
+
+  - uses: go/build
+    with:
+      packages: ./cmd/gce-pd-csi-driver/main.go
+      ldflags: |
+        -X main.version=$(git describe --long --tags --match='v*' --dirty 2>/dev/null || git rev-list -n1 HEAD)
```

Comment:
can't we just use ${{package.version}} here?

Reply:
Yes, we can. This is what upstream was using. It contains a mix of version and commit if done in dev but we are not doing that so we can proceed with using `v${{package.version}}`

but agree we can update it looking at logs. 

```bash
$ docker run --rm registry.k8s.io/cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver:v1.15.1 -v 10
I1031 12:55:34.509737       1 main.go:114] Operating compute environment set to: production and computeEndpoint is set to: <nil>
I1031 12:55:34.510037       1 main.go:123] Sys info: NumCPU: 11 MAXPROC: 1
I1031 12:55:34.510043       1 main.go:128] Driver vendor version v1.15.1
```